lib/rpm_package.py

```diff
@@ -28,6 +28,10 @@
 
 CONF = config.get_config().CONF
 LOG = logging.getLogger(__name__)
+PACKAGE_METADATA_TO_RPM_MACRO_MAPPING = {
+    ("sources", 0, "git", "commit_id"): "git_commit_id",
+    ("sources", 0, "svn", "commit_id"): "svn_revision",
+}
```

Comment:
- I'd document that we are assuming that the first source is the main one
- Shouldn't we support Mercurial too?